src/custom/components/Footer/index.tsx

```diff
@@ -12,18 +13,18 @@ const Wrapper = styled.div`
   ${({ theme }) => theme.flexRowNoWrap}
   align-items: center;
   justify-content: space-evenly;
-  margin: auto 6rem 0 2rem;
-
+  margin: auto 96px 0 32px;
   width: 100%;
 
   ${({ theme }) => theme.mediaWidth.upToMedium`
```

Comment:
there's no media query here so remove this right?

Reply:
actually you should display none the `FooterVersion` and children

Reply:
Question: What if we keep showing it, and allow the user to scroll down and see those contents?

Otherwise, in case we'll use the click wrap in the future, we'll need to load the Clickwrap probably higher up (App container).

Reply:
just add the FooterVersion and children in a wrapper that is hidden on media but keep the clickwrap visible

Reply:
Ah that's right, added now.